bundle/edu.gemini.itc/src/main/scala/edu/gemini/itc/base/Recipe.scala

```diff
@@ -4,7 +4,7 @@ import java.util
 
 import edu.gemini.itc.shared._
 
-import scala.collection.JavaConversions
+import scala.collection.JavaConversions._
```

Comment:
Here in Scala code I think we're better off with the explicit pimping conversions of `JavaConverters`.